db/migrate/20200916194032_add_status_to_merchantts.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToMerchantts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :merchants, :status, :string, default: "enabled"
```

Comment:
Your class name has two "t"s in it. Looks like it doesn't affect it after you made the change to line 3.

Reply:
Ugh fat fingers